docs/_includes/exec.md

```diff
@@ -84,7 +85,7 @@ Compose an `sbatch` script that encapsulates resource requests, module loading a
 
 module purge
 module load java
-/home/$USER/bin/nextflow run labsyspharm/mcmicro --in /n/scratch3/users/${USER:0:1}/$USER/exemplar-001 -profile O2 -w /n/scratch3/users/${USER:0:1}/$USER/work
+/home/$USER/bin/nextflow run labsyspharm/mcmicro --in /n/scratch3/users/${USER:0:1}/$USER/exemplar-001 -profile O2 -w /n/scratch3/users/${USER:0:1}/$USER/work -with-report "/n/groups/lsp/mcmicro/reports/$USER-$EPOCHREALTIME.html"
```

Comment:
Would it make sense to break this up into multiple lines, since it just keeps getting longer?